inst/extdata/run_seeker_array.R

```diff
@@ -1,6 +1,20 @@
 cArgs = commandArgs(TRUE)
-yamlPath = cArgs[1L]
-parentDir = cArgs[2L]
+if (length(cArgs) == 2) {
```

Comment:
Too complicated. I still think the R script should just load a yaml file and the shell script should not change. I don't think anything in the reproducibility vignette has to change. Only one line in run_seeker_array.R needs to change.

Reply:
As requested, please restore the reproducibility vignette to the version on master.

Reply:
Done in https://github.com/hugheylab/seeker/pull/34/commits/1774d453dc5f18373968ee483886a7bfe56b7f10